packages/dds/tree/src/simple-tree/api/storedSchema.ts

```diff
@@ -50,7 +51,7 @@ import { ViewSchema } from "./view.js";
  */
 export function extractPersistedSchema(schema: ImplicitFieldSchema): JsonCompatible {
 	const stored = toStoredSchema(schema);
-	return encodeTreeSchema(stored);
+	return encodeTreeSchema(stored, SchemaCodecVersion.v1);
```

Comment:
fine for now, but we should really consider a follow-up allowing people to choose which SharedTreeFormatVersion they want to use to get an idea of what their persisted schema data looks like.